Cookbook/README.md

```diff
@@ -0,0 +1,38 @@
+iOS Cookbook 👩‍🍳
+ ====================================
+
+## I am here because I want to ...
+
+##### ... learn about our architecture 🏠
+
+* [Implementing features with ReactiveFeedback](http://ilya.puchka.me/implementing-features-with-reactivefeedback/)
+* [Global `Current` struct to hold global dependencies](./Proposals/ControlTheWorld.md)
+* [Posting Analytics Events to Snowplow](./Technical-Documents/SnowplowHowTo.md)
+* [Use builders as functions](./Technical-Documents/BuildersToFunctions.md)
+
+##### ... build my feature with Bento 🍱
+
+* [Generalized Screen Lifecycle Observation](./Proposals/ScreenLifecycleObservation.md)
+* [Design Library](./Technical-Documents/DesignLibrary.md)
+
+##### ... learn how the iOS team works 🛠
+
+* [Code Owners - Restoring developer's efficiency](./Proposals/CODEOWNERS.md)
+* [iOS Team style guide](./Style-guide/)
+* [New hires checklist](./Technical-Documents/NewHiresChecklist.md)
+* [How to make a proposal](./Proposals/Template_Proposal.md)
+
+##### ... test my app 🌳
+
+* [Replace `iOSSnapshotTestCase` library with `SnapshotTesting`](./Proposals/SnapshotTesting.md)
+* [Automation Identifiers](./Technical-Documents/AutomationIdentifiers.md)
+
+##### ... learn functional programming 🚀
+
+* [Introduce "fish" operator (Kleisli composition)](./Proposals/Fish_Operator.md)
+
+##### ... learn about other topics 🏔
+
+* [Strings and Localisation](./Technical-Documents/Lokalise.md)
+* [Outstanding Technical Debt and Legacy Code](./Technical-Documents/TechnicalDebt.md)
+    * [BBAddAdditionalPatientInformationViewControllerV2](./Technical-Documents/TechnicalDebt.md)
```

Comment:
```suggestion
* [BBAddAdditionalPatientInformationViewControllerV2](./Technical-Documents/TechnicalDebt.md)
```

Reply:
I added the indentation on purpose since it's part of legacy.